modules/ipi-install-network-requirements.adoc

```diff
@@ -15,13 +15,13 @@ Installer-provisioned installation of {product-title} involves several network r
 +
 In {product-title} 4.3, when deploying using the `provisioning` network, the first NIC on each node, such as `eth0` or `eno1`, must interface with the `provisioning` network.
 +
-In {product-title} 4.4 and later releases, you can specify the provisioning network NIC with the `provisioningNetworkInterface` configuration setting.
+In {product-title} 4.4 and later releases, the `bootMacAddress` field is used to calculate the IP address for Ironic to bind to. The `provisioningNetworkInterface` configuration field is optional.
 
 - `baremetal`: The `baremetal` network is a routable network.
 +
 In {product-title} 4.3, when deploying using the `provisioning` network, the second NIC on each node, such as `eth1` or `eno2`, must interface with the `baremetal` network.
 +
-In {product-title} 4.4 and later releases, you can use any NIC order to interface with the `baremetal` network, provided it is the same NIC order across worker and control plane nodes and not the NIC specified in the `provisioningNetworkInterface` configuration setting for the `provisioning` network.
+In {product-title} 4.4 and later releases, you can use any NIC order to interface with the `baremetal` network, provided it is the same NIC order across worker and control plane nodes. If using the `provisioningNetworkInterface` configuration field, ensure you are not using the specified NIC.
```

Comment:
Maybe:
...configuration field, ensure that you are using...

Reply:
I also found this confusing and thought it was supposed to be "are using", but the original documentation says "and not the NIC specified in the `provisioningNetworkInterface`...". I will leave it and see what QE thinks.